### PR TITLE
Unifying names of tryDest*

### PR DESCRIPTION
**Pull request by forki**
The naming here always confused me. I'm trying to unify this

### CODE REVIEW

File: src/fsharp/TastOps.fs [hunk 1]
```diff
@@ -816,9 +816,9 @@ let tryAppTy g ty = ty |> stripTyEqns g |> (function TType_app(tcref, tinst) ->
 let destAppTy g ty = ty |> stripTyEqns g |> (function TType_app(tcref, tinst) -> tcref, tinst | _ -> failwith "destAppTy")
 let tcrefOfAppTy g ty = ty |> stripTyEqns g |> (function TType_app(tcref, _) -> tcref | _ -> failwith "tcrefOfAppTy") 
 let argsOfAppTy g ty = ty |> stripTyEqns g |> (function TType_app(_, tinst) -> tinst | _ -> [])
-let tryDestTyparTy g ty = ty |> stripTyEqns g |> (function TType_var v -> ValueSome v | _ -> ValueNone)
-let tryDestFunTy g ty = ty |> stripTyEqns g |> (function TType_fun (tyv, tau) -> ValueSome(tyv, tau) | _ -> ValueNone)
-let tryDestAppTy g ty = ty |> stripTyEqns g |> (function TType_app(tcref, _) -> ValueSome tcref | _ -> ValueNone)
+let tryTyparTy g ty = ty |> stripTyEqns g |> (function TType_var v -> ValueSome v | _ -> ValueNone)
+let tryFunTy g ty = ty |> stripTyEqns g |> (function TType_fun (tyv, tau) -> ValueSome(tyv, tau) | _ -> ValueNone)
+let tryTcrefOfAppTy g ty = ty |> stripTyEqns g |> (function TType_app(tcref, _) -> ValueSome tcref | _ -> ValueNone)
 let tryDestAnonRecdTy g ty = ty |> stripTyEqns g |> (function TType_anon (anonInfo, tys) -> ValueSome (anonInfo, tys) | _ -> ValueNone)
```
Comment (KevinRansom) [inline src/fsharp/TastOps.fs]: @forki is this name less confusing than the others?

Comment (forki) [reply src/fsharp/TastOps.fs]: No. That one will go in next round. This is a battle I'm fighting for many years. After many unmerged PRs I'm trying to do smaller steps ;-)

Comment (forki) [reply src/fsharp/TastOps.fs]: No. That one will go in next round. This is a battle I'm fighting for many years. After many unmerged PRs I'm trying to do smaller steps ;-)

**Comment by forki** [comment]
In some cases if gives you the whole payload and in others only the tcref. It's inconsistent

**Comment by forki** [comment]
@TIHan just compare destAppTy to tryDestAppTy

**Comment by forki** [comment]
done

**Comment by KevinRansom** [comment]
@forki , thanks mate for taking care of this.

Kevin

**Pull request merged**